### ISSUE TITLE
[Serious] Focus indicator is not visible

### ISSUE THREAD

**Issue by ntallman** [APTrust/registry#100]
## Issue Details

| | |
|---|---|
| **Source Issue ID** | 87581 |
| **Checkpoint** | 2.4 Navigable › Focus Visible (2.4.7.a) |
| **Impact** | Serious |
| **Standards** | wcag2a, wcag2aa, wcag21a, wcag21aa, wcag22a, wcag22aa |
| **Test Area** | 05 Add New User Workflow |
| **Test Case Name** | AP Trust Registry (Demo) |
| **URL** | <https://demo.aptrust.org/users> |
| **Environment** | Production |
| **Platform** | Windows | Chrome | Firefox |
| **Assistive Technology** | JAWS | NVDA | Magnifier |
| **Method** | Manual |
| **Digital Asset Type** | Desktop Web |
| **Release** | 4.2026 |
| **Product** | Website |
| **Date Reported** | 05/06/26 at 12:04 pm |
| **Unit Type** | Page |
| **Issue Type** | Accessibility |
| **Reported By** | a3c.access@gmail.com |

## Description

The focus indicator (e.g. border or dotted underline / background etc.) is not visible as one tabs through the page.
Pattern: the green buttons, for example, the Add New User button, has no visual focus indicator to tell users that the item has been focused. See Managing Users User Experience Video at 1:00.

## Recommendation

Rule : 
All focusable elements MUST have a visual focus indicator when in focus.

Focusable elements SHOULD have enhanced visual focus indicator styles.

How To Fix: 
Common keyboard focus indicator solutions include:
1. A 2px box around the focused element
2. A change in the background color of the focused element
3. The addition of an icon, such as an arrow, next to a menu item
4. The addition of a thick line under or next to the focused element
5. A distinctive change to the text of the component such as making it bold and/or underlined
6. Use the default browser focus indicator. NOTE: Since each browser has its own default focus indicator style, check each of the major browsers (Chrome, Firefox, Edge, IE, Safari) to ensure the default focus indicator is visible as you tab through the page. 

A keyboard focus indicator can take different forms; it does not have to be a boring box. But the goal is to provide a difference in contrast between a component's default and focused states of at least 3 to 1 or another distinctive visual change.

Background: 
When a visible keyboard focus indicator is not provided, sighted keyboard users will have no idea which link or control has focus making it extremely difficult, if not impossible, to interact with the content. Browsers provide default focus indicators for natively focusable elements, but these may be very difficult to see depending on the color of the control and the page background. In addition, custom elements often have no visible focus indicator at all. Ideal focus indicators are designed to provide good contrast with links and controls and their backgrounds. Focus indicators with good contrast make it much easier to track focus as a keyboard user navigates through the page.

## Source Code

```
<a class="button is-success mr-6 is-not-underlined" href="/users/new?institution_id=81">
      Add new user
    </a>
```

## Screenshots

![05_focus visible](https://raw.githubusercontent.com/APTrust/registry/master/docs/accessibility-audit/screenshots/87581/05_focus_visible.png)
